deploy/olm-catalog/portworx/23.7.0/portworx-certified.clusterserviceversion.yaml

```diff
@@ -217,6 +217,14 @@ spec:
                         values:
                         - portworx-operator
                     topologyKey: kubernetes.io/hostname
+                nodeAffinity:
+                  requiredDuringSchedulingIgnoredDuringExecution:
+                    - labelSelector:
```

Comment:
We need to fix this. This structure is incorrect as it was in master. I guess we are missing a cherrypick for this.

This has broken the travis build in 23.7.0 branch.

Reply:
Yes that cherrypick was missed, raised PR https://github.com/libopenstorage/operator/pull/1231